packages/react/src/Icon/Icon.tsx

```diff
@@ -1,4 +1,4 @@
-import React, {type SVGAttributes} from 'react'
+import React, {cloneElement, isValidElement, type ReactElement, type SVGAttributes} from 'react'
```

Comment:
Nit, but non-blocking. I prefer to not destructure React methods to help code readability on long component files. Makes it obvious it's not a local function. Think it's fine for types though.